terraform/modules/matrix-service/lambdas/notifications_lambda.tf

```diff
@@ -0,0 +1,99 @@
+resource "aws_iam_role" "matrix_service_notifications_lambda" {
+  name = "matrix-service-notifications-daemon-${var.deployment_stage}"
+
+  assume_role_policy = <<POLICY
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "lambda.amazonaws.com"
+      },
+      "Effect": "Allow",
+      "Sid": ""
+    }
+  ]
+}
+POLICY
+}
+
+resource "aws_iam_role_policy" "matrix_service_notifications_lambda" {
+  name = "matrix-service-notifications-daemon-${var.deployment_stage}"
+  role = "${aws_iam_role.matrix_service_notifications_lambda.name}"
+  policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "LogsPolicy",
+      "Effect": "Allow",
+      "Resource": [
+        "arn:aws:logs:${var.aws_region}:${var.account_id}:log-group:/aws/lambda/dcp-matrix-service-notifications-${var.deployment_stage}",
+        "arn:aws:logs:${var.aws_region}:${var.account_id}:log-group:/aws/lambda/dcp-matrix-service-notifications-${var.deployment_stage}:*:*"
+      ],
+      "Action": [
+        "logs:CreateLogGroup",
```

Comment:
Does this work? I thought logs:CreateLogGroup needs * resource path?

Reply:
Yup, I'm getting logs in CloudWatch